.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,69 @@
+name: Bug Report
+description: Issues relating to code/workflow errors
+title: "[BUG] <Title of Bug>"
+labels: "bug"
+body:
+
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+
+- type: textarea
+  attributes:
+    label: Conda environment
+    description: Add conda environment used.
+  validations:
+      required: true
+
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: false
+
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A concise description of what you expected to happen.
+  validations:
+    required: false
+
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: How do you trigger this bug? Please walk us through it step by step.
+    placeholder: |
+      1. Running this command...
+      2. Produces this result...
+  validations:
+    required: false
+
+- type: input
+  attributes:
+    label: Operating System 
```

Comment:
Can we change this one to a dropdown? I feel like knowing the exact version of MacOS or whatever isnt going to be where I start debugging haha. Maybe something like 

```yaml 
  - type: dropdown
    id: os
    attributes:
      label: Operating System
      description: What operating system are you running?
      options:
        - Linux
        - MacOS
        - Windows
    validations:
      required: true
```

Reply:
Yep can do!